src/main/java/com/iexec/common/chain/eip712/EIP712Utils.java

```diff
@@ -91,4 +95,14 @@ static String encodeByteArray(byte[] byteArray) {
      *
      */
 
+    public static String buildAuthorizationToken(EIP712Challenge eip712Challenge, String walletAddress, ECKeyPair ecKeyPair) {
```

Comment:
Wouldn't it be a better implementation to move this method to `EIP712Entity` with a name like `buildAuthorizationTokenFromCredentials(Credentials credentials)` ?
We can then extract the ecKeyPair and the walletAddress from the credentials and use the getHash and signMessage methods from EIP712Entity.

I would prefer something like this instead of an util class with a single static method.